templates/helpers/jsesc.js

```diff
@@ -0,0 +1,5 @@
+var jsesc = require('jsesc');
+
+module.exports = (value) => jsesc(
+    value.replace(/[\r\n]{1,2}/g, '')
```

Comment:
I think this needs `.replace(/<\/script/g, '<\\/script>')` since the output is placed inside a `<script>` in HTML. 

My earlier question remains, though: why `.replace(/[\r\n]{1,2}/g, ')'`?